browser-test/src/applicant/northstar_upsell.test.ts

```diff
@@ -60,6 +62,54 @@ test.describe('Upsell tests', {tag: ['@northstar']}, () => {
     await validateApplicationSubmittedPage(
       page,
       /* expectRelatedProgram= */ true,
+      /* expectedDownloadApplicationLink= */ true,
+      applicantQuestions,
+    )
+
+    await test.step('Validate screenshot and accessibility', async () => {
+      await validateScreenshot(
+        page,
+        'upsell-north-star-download',
+        /* fullPage= */ true,
+        /* mobileScreenshot= */ true,
+      )
```

Comment:
Question (blocking): Is it necessary to have another set of screenshots just to validate the presence of the download button?

I think `validateApplicationSubmittedPage` already checks for its existence. We have a lot of screenshot noise so I think it's better to *not* add another call to `validateScreenshot` for this test case. What do you think?

Related guidance: https://github.com/civiform/civiform/wiki/Testing#how-to-add-new-screenshots

"Use screenshots judiciously. They should be used as a second layer of verification, rather than the primary method, which should be using assertions and checking for elements on the page. Not every test needs a screenshot, and we should generally have one per page. We may want additional screenshots if content is significantly changing on the page during a particular test, and consider taking a screenshot of the particular element you are looking at. Don't take a screenshot that looks basically the same as another."

Reply:
i'm ok with only having 1 screenshot for the view with the most elements (that would be WITH download), which follows the guidance.